dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/utils/JSONUtilsTest.java

```diff
@@ -199,8 +203,6 @@ public void testToMap() {
 
         Assert.assertNull(JSONUtils.toMap("3"));
```

Comment:
It seems you need to remove this test method

Reply:
> It seems you need to remove this test method

I removed three unused imports here. Also why do we remove this testToMap() method?

Reply:
> It seems you need to remove this test method



> It seems you need to remove this test method

is it because it overlaps with `toMap` test method?

Reply:
Since we have remove the `JSONUtils.toMap` method, so this will compile failed.

Reply:
oh okay, I do think we can get rid of toMap by looking back to discussions. will do that and commit tomorrow. (It's midnight here in my time zone)

Reply:
> Since we have remove the `JSONUtils.toMap` method, so this will compile failed.

oh okay, I do think we can get rid of toMap by looking back to discussions. will do that and commit tomorrow. (It's midnight here in my time zone)